src/console/spcm_control/rx_device.py

```diff
@@ -320,14 +371,14 @@ def _gated_timestamps_stream(self):
                         else:
                             gate_data = rx_data[index_0 : index_0 + int(total_bytes / 2)]
 
-                        pre_trigger_cut = (self.pre_trigger - 1) * self.num_channels.value
```

Comment:
I forgot why we deleted -1. I should doublecheck this part.

Reply:
I think this was just a test from my side, without the -1 it should be fine. 
But we should definetly double-check this in general. I suggest to solve this in PR #29